diff --git a/systemd/cloud-init.service b/systemd/cloud-init.service
index fb3b918..2d8e97c 100644
--- a/systemd/cloud-init.service
+++ b/systemd/cloud-init.service
@@ -1,6 +1,5 @@
 [Unit]
 Description=Initial cloud-init job (metadata service crawler)
-DefaultDependencies=no
 Wants=cloud-init-local.service
 Wants=sshd-keygen.service
 Wants=sshd.service
@@ -9,7 +8,6 @@ After=networking.service
 Before=network-online.target
 Before=sshd-keygen.service
 Before=sshd.service
-Before=sysinit.target
 Before=systemd-user-sessions.service
 Conflicts=shutdown.target
 
